### DISCUSSION TITLE
Validar comportamiento de selector de survey+api

### DISCUSSION THREAD

**Discussion by jdurandpaz** [Cotalker/documentation#494] [2. Questions and Answers]
Estimados,

Me gustaría saber si el comportamiento esperado de los survey+api es el siguiente:

1. Se completan campos en el formulario

![image](https://user-images.githubusercontent.com/50208672/138106130-a2758c26-470d-4187-823b-b59172acc7d8.png)

2. Se modifica uno de los survey+api, luego de esto queda un elemento flotando con la última selección que desaparece si se hace click en algun lugar de la pantalla

![image](https://user-images.githubusercontent.com/50208672/138105781-ef170bdc-68b4-40f7-86bd-0437070d48e5.png)

**Comment by acedward** [comment]
¿Pasa en todos los survey+api? Nunca habia visto esto (claramente es un bug si queda el elemento abierto por siempre)

**Reply by jdurandpaz** [reply]
Según valide con el equipo, también les pasa en otros survey+api de otras companies.
